packages/plugins/i18n/server/bootstrap.js

```diff
@@ -8,11 +8,17 @@ const registerModelsHooks = () => {
     .map((contentType) => contentType.uid);
 
   if (i18nModelUIDs.length > 0) {
+    // TODO V5 : to remove ?
+    // Should this code exist? It's putting business logic on the query engine
+    // whereas it should maybe stay on the entity service layer ?
```

Comment:
I do think this code belongs in the i18n plugin. As it is about assigning locales.

Maybe this bit of code could be turned into a service if we see it appropiate, but I do think it should stay in v5.


This is a good way of decoupling logic from core, but  I also think we could:
- document this side effects on the entities. 
- standarize where we apply change effects like this (or extending controllers and services for instance)
 
If we start relying more and more on side effects like this it could become tricky to track whats changing what.

Reply:
Sorry I didn't mean to move the code elsewhere. My question was about just deleting it. I wonder if it's not crossing a boundary to do i18n logic in the database hooks. Adding a default locale is already managed at the entity-service level (with `packages/plugins/i18n/server/services/entity-service-decorator.js`).